aiven/client/common.py

```diff
@@ -1,2 +1,2 @@
 # Copyright (c) 2021 Aiven, Helsinki, Finland. https://aiven.io/
-UNDEFINED = object()
+UNDEFINED: object = object()
```

Comment:
Probably not really needed. Did `mypy` complain about it? I think it has pretty good type inference for cases like this.